frontend/src/metabase/dashboard/components/DashCard/DashCardVisualization.tsx

```diff
@@ -62,7 +62,7 @@ interface DashCardVisualizationProps {
 
   isAction: boolean;
   isPreviewing: boolean;
-  isEmbed: boolean;
+  isEmbedded: boolean;
```

Comment:
Seems we have more usages of `isEmbedded` naming, so after this PR we don't have the `isEmbed` naming in code.